tests/unit/siswrap_handlers_tests.py

```diff
@@ -1,26 +1,35 @@
 import pytest
-from siswrap.siswrap import *
-from siswrap.siswrap_ws import *
-from siswrap.configuration import *
 import tornado.web
 import jsonpickle
+from arteria import *
+from arteria.configuration import ConfigurationService
+from siswrap.app import *
+from siswrap.handlers import *
+from siswrap.wrapper_services import *
 
-# Some unit tests for siswrap.siswrap_ws
+
+# Some unit tests for siswrap.handlers
 
 API_URL = "/api/1.0"
 
 
 @pytest.fixture
 def app():
-    return SisApp.create_app(debug=True)
-
+    config_svc = ConfigurationService(app_config_path="./config/app.config")
+    process_svc = ProcessService(config_svc)
+    args = dict(process_svc=process_svc, config_svc=config_svc)
+    app = tornado.web.Application([
```

Comment:
Maybe it would be a good idea to break out the routes so that you can import them?